packages/mobile/src/escrow/EscrowedPaymentLineItem.tsx

```diff
@@ -1,24 +1,33 @@
 import fontStyles from '@celo/react-components/styles/fonts'
 import * as React from 'react'
+import { Trans, withNamespaces, WithNamespaces } from 'react-i18next'
 import { StyleSheet, Text } from 'react-native'
 import { EscrowedPayment } from 'src/escrow/actions'
+import { CURRENCIES, CURRENCY_ENUM } from 'src/geth/consts'
 import { divideByWei, getCentAwareMoneyDisplay } from 'src/utils/formatting'
 
 interface Props {
   payment: EscrowedPayment
 }
 
-export default function EscrowedPaymentLineItem(props: Props) {
-  const { amount, message, recipientPhone } = props.payment
+function EscrowedPaymentLineItem(props: Props & WithNamespaces) {
+  const { amount, recipientPhone } = props.payment
   return (
     <Text numberOfLines={1} ellipsizeMode="middle" style={styles.oneLine}>
-      <Text style={[fontStyles.subSmall]}>
-        {recipientPhone} - {message}
-      </Text>
-      <Text style={[fontStyles.subSmall, fontStyles.semiBold]}>
-        {' '}
-        ${getCentAwareMoneyDisplay(divideByWei(amount.toString()))}
-      </Text>
+      <Trans
+        i18nKey="escrowPaymentNotificationLine"
+        // @ts-ignore tOptions prop is missing in type bindings, but exists in the implementation
+        tOptions={{ context: !recipientPhone ? 'missingRecipientPhone' : null }}
+        values={{
+          amount:
+            CURRENCIES[CURRENCY_ENUM.DOLLAR].symbol +
+            getCentAwareMoneyDisplay(divideByWei(amount.toString())),
+          recipientPhone,
+        }}
+      >
+        <Text style={fontStyles.subSmall}>{{ recipientPhone }} for </Text>
+        <Text style={[fontStyles.subSmall, fontStyles.semiBold]}>{{ amount }}</Text>
```

Comment:
You don't have to fix this but just an FYI, we would generally prefer to have a style defined below which does something like this:
```
{
...fontStyles.subSmall,
...fontStyles.semiBold
}
```

But it's really nbd. Just an FYI on the convention we're _roughly_ trying to keep to elsewhere